app/assets/stylesheets/variables/colors.scss

```diff
@@ -17,3 +17,4 @@ $black: #2b2b2b;
 $green-light: #d6e9c6;
 $green-dark: #3c763d;
 $light: #f5f5f5;
+$breadcrumb-gray: #5e5e5e;
```

Comment:
For color variables we like to describe the color rather than the intended use, because if I want to then use this same color in another place, like a footer or something, it reads a bit awkward.

Reply:
Okay I think we can remove this line then and we're good to go.